src/SmartComponents/Drift/DriftCard.js

```diff
@@ -0,0 +1,174 @@
+import './DriftCard.scss';
+
+import * as AppActions from '../../AppActions';
+import * as ActionTypes from '../../AppConstants';
+
+import React, { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+import { ExpandableCardTemplate } from '../../PresentationalComponents/Template/ExpandableCardTemplate';
+import { TemplateCardBody, TemplateCardHeader } from '../../PresentationalComponents/Template/TemplateCard';
+import { Flex, FlexItem } from '@patternfly/react-core/dist/esm/layouts';
+import { DriftDropDown } from './DriftDropDown';
+import messages from '../../Messages';
+import SortUpIcon from '@patternfly/react-icons/dist/esm/icons/sort-up-icon';
+import {
+    TextContent,
+    Divider,
+    DataList,
+    DataListItem,
+    DataListItemRow,
+    DataListCell,
+    DataListItemCells,
+    DataListWrapModifier,
+    Button,
+    TextVariants,
+    Text
+} from '@patternfly/react-core';
+import { DriftEmptyState } from './DriftEmptyState';
+import Loading from '../../PresentationalComponents/Loading/Loading';
+import { useDispatch } from 'react-redux';
+import routerParams from '@redhat-cloud-services/frontend-components-utilities/RouterParams';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { useCallback } from 'react';
+
+const DriftCard = ({ driftEvents, driftEventFetchStatus }) => {
+
+    const intl = useIntl();
+    const dispatch = useDispatch();
+    let currentItemIdx = 3;
```

Comment:
This won't trigger re-render. You should use `useState`
```suggestion
    const [activeDrift, setActiveDrift] = useState({
      id: 'days-7',
      description: intl.formatMessage(messages.driftDropDown7days),
      startDate: getDate(7),
      endDate: getDate(0)
    });
```